src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,21 @@
+@import "../utils/variables";
+@import "../utils/_mixins";
```

Comment:
let's import all files in one general index.scss or main.scss

Reply:
delete redundant import from files with styles for separate blocks
you've already imported them in one main css file